go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
 	github.com/cenkalti/backoff/v4 v4.3.0
 	github.com/confluentinc/confluent-kafka-go/v2 v2.8.0
-	github.com/databricks/databricks-sql-go v1.7.0
+	github.com/databricks/databricks-sql-go v1.6.1
```

Comment:
Was the version bumped in release 1.47 ?

Reply:
Yes. https://github.com/rudderlabs/rudder-server/pull/5709